SteamProject/MySteam/Category.cpp

```diff
@@ -33,24 +62,43 @@ bool FCategory::AddGame(const FGame NewGame)
     return false;
 }
 
+//-------------------------------------------------------------------------------------------------------------
+
 bool FCategory::HasSpaceForGames() const
 {
     return GamesAmount < MaxGamesAmount;
 }
 
+//-------------------------------------------------------------------------------------------------------------
+
 bool FCategory::IsValid() const
 {
     return !CategoryName.empty();
 }
 
+//-------------------------------------------------------------------------------------------------------------
+
 void FCategory::ShowAllGames() const
 {
-    for (FGame LoopGame : GameArray)
+    for (const FGame& LoopGame : GameArray)
     {
         if (LoopGame.IsValid())
         {
-            std::cout << LoopGame.GetGameName() << "\t" << LoopGame.GetStudioName() << "\t" << LoopGame.GetReleaseDate() << std::endl;
+            LoopGame.PrintGame();
```

Comment:
Good job on turning this into a function of the game! I think it works a lot better for reading the logic of this function. Even if it's just one line, I think the possible reusability of it makes it worth it.